src/models/sequential.cpp

```diff
@@ -70,7 +70,7 @@ void sequential_model::write_proto(lbann_data::Model* proto) {
   model::write_proto(proto);
   //Add layers
   if (m_comm->am_world_master()) {
-    proto->set_name(name());
+    proto->set_name(type());
```

Comment:
It looks like the proto code needs to change the function name from set_name to set_type to be consistent.